samples/python/bulk_edit_issues.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import json
+import csv
+
+DOMAIN = "<DOMAIN>"
+API_KEY = "<api-key>"
+ENDPOINT = "https://api.helpshift.com/v1/" + DOMAIN + "/issues"
+
+PAGE_SIZE = 1000 #max value
+BULK_CHUNK = 5000 #max value
+
+# Add to `api_params` to edit issues as per your requirement.
+
+def bulk_edit_issue(issues):
+    print ("Triggering Bulk Action for %d issues starting with id %s and ends with id %d" %((len(issues)), issues[0], issues[-1]))
+    api_params = "/?issue-ids=" + json.dumps(issues) + "&status=Resolved" #example
```

Comment:
this will still result in 431.

Reply:
This is taken as an improvement. Will be done in a future commit.